packages/collaboration/comments/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# @udecode/plate-comments
+
+## 1.0.0
```

Comment:
> Overall very smart and complete work.
> 
> My first request is to use a thread leaf instead of an inline element:
> 
> * it will enable us to overlap threads
> * use `thread_<id>`  prop to store the thread
> * slate can split an element into multiple leaves so we'll need to set/unset nodes to all siblings with the same thread id so all the thread leaves are in sync
> * later we can extend the feature to inline elements and blocks with the same structure, queries and transforms
> * to see an example, Slite is using that strategy
> 
> I already did a plugin with the same structure so I'll share a gist we can reuse:
> 
> https://gist.github.com/zbeyens/dfcf97a8cb2af991d91ca506d500b74a
> 
> My second request is to use a [zustood store](https://gist.github.com/zbeyens/dfcf97a8cb2af991d91ca506d500b74a) (as we do for core and combobox plugin) to store the active thread, hover thread, all the threads, etc.
> 
> What do you think?

Thanks. Sounds reasonable.